kb/prometheus-grafana-setup-cloud.md

```diff
@@ -0,0 +1,269 @@
+4. In the **HTTP** section, enter your Prometheus endpoint in the URL field.
+   If running Prometheus locally as described in the examples in this article, enter `http://localhost:9090`.
+5. For this example, enable **Skip TLS Verify** in the **Auth** section.
+6. Select **Save and test** to ensure that the data source is working properly.
+
+If you see issues in setting this data source, check whether the endpoints set in your SDKs are showing metrics.
+If you don’t see your SDK metrics at the scrape endpoints defined, check whether your Workers and Workflow Executions are running.
+If you see metrics on the scrape endpoints, but Prometheus shows your targets are down, then there is an issue with connecting to the targets set in your SDKs.
+Verify your Prometheus configuration and restart Prometheus.
+
+If you’re running Grafana as a container, you can set your SDK metrics Prometheus data source in your Grafana configuration.
+See the [example Grafana configuration](/kb/prometheus-grafana-setup) described in the Prometheus and Grafana setup for open-source Temporal Cluster KB article.
+
+## Grafana dashboards setup
+
+To set up your dashboards in Grafana, either use the UI or configure them in your Grafana deployment.
+
+In this article, we will configure our dashboards using the UI.
+
+1. Go to **Create&nbsp;> Dashboard**, and add an empty panel.
+2. On the **Panel configuration** page, in the **Query** tab, select the "Temporal Cloud metrics" or "Temporal SDK metrics" data source
+   that we configured in the previous section.
+   If you want to add multiple queries that involve both data sources, select `–Mixed–`.
+3. Add your metrics queries:
+   - For Temporal Cloud metrics, expand the **Metrics browser** and select the metrics you want to see.
+     You can also select associated labels and values to sort the data on the query.
+     The documentation on [Cloud metrics](/cloud/how-to-monitor-temporal-cloud-metrics#available-performance-metrics) lists metrics emitted from Temporal Cloud.
+   - For Temporal SDK metrics, expand the **Metrics browser** and select the metrics you want to see.
+     A list of metrics on Worker performance are described in [Developer's Guide - Worker performance](/application-development/worker-performance).
+     All metrics related to SDKs are described in the [SDK metrics](/references/sdk-metrics) reference.
+4. You should see the graph show data based on the queries you have selected.
+   Note that for SDK metrics to show, you must have some Workflow Execution data and running Workers.
+   If you do not see any metrics data from the SDK, run your Worker and Workflow Executions and monitor your dashboard.
+
+Temporal has a repository with some community-driven example dashboards for [Temporal Cloud](https://github.com/temporalio/dashboards/tree/master/cloud) and [Temporal SDKs](https://github.com/temporalio/dashboards/tree/master/sdk) that you can use and customize for your own requirements.
```

Comment:
Can we put this as a kind of `;tldr` at the top and is very LOUD. Users will refer to this often and it is pretty hidden down here